flutter/lib/default_integrations.dart

```diff
@@ -84,6 +84,49 @@ Integration runZonedGuardedIntegration(
   return integration;
 }
 
+Integration deviceInfosIntegration(
+  SentryOptions options,
+  MethodChannel channel,
+) {
+  Future<void> integration(Hub hub, SentryOptions options) async {
+    try {
+      final Map<String, dynamic> infos =
+          Map<String, dynamic>.from(await channel.invokeMethod('deviceInfos'));
+
+      if (infos['contexts'] != null) {
+        options.addEventProcessor((event, dynamic hint) {
+          event = event.copyWith(
+            contexts: Contexts.fromJson(
+              Map<String, dynamic>.from(infos['contexts'] as Map),
+            ),
+          );
+          return event;
+        });
+      }
+
+      if (infos['integrations'] != null) {
+        final integrations = List<String>.from(infos['integrations'] as List);
+        options.addEventProcessor((event, dynamic hint) {
```

Comment:
looks a bit overkill having 2 event processors for the same thing, can we merge them?

Reply:
✅